### PR TITLE
Adjust and parametrize ZMON components

### PR DESCRIPTION
**Pull request by mohabusama**
Adjust and parametrize ZMON components

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -86,7 +86,11 @@ zmon_worker_count: "0"
 {{else if eq .Environment "production"}}
 logging_agent_enabled: "true"
 zmon_agent_replicas: "1"
+zmon_agent_mem: "500Mi"
```
Comment (aermakov-zalando) [inline cluster/config-defaults.yaml]: You can just define it once outside the `if`, it'd be easier to keep the settings in sync.

Comment (mohabusama) [reply cluster/config-defaults.yaml]: done

**Comment by aermakov-zalando** [comment]
👍

**Comment by szuecs** [comment]
:+1:

**Pull request merged**